game.js

```diff
@@ -0,0 +1,359 @@
+rediv = document.querySelector('.rebtn');
+rebtn.onclick = function() {
+    location.reload();
+}
+document.querySelector('#onebtn').onclick = function() {
+    document.querySelector('.oneopt').classList.remove('hide');
+    document.querySelector('.oneopta').classList.remove('hide');
+    document.querySelector('.twobtn').classList.add('hide');
+    document.querySelector('.onebtn').classList.add('hide');
+}
+var convert = new Map();
+convert.set(1,'.b1');
+convert.set(2,'.b2');
+convert.set(3,'.b3');
+convert.set(4,'.b4');
+convert.set(5,'.b5');
+convert.set(6,'.b6');
+convert.set(7,'.b7');
+convert.set(8,'.b8');
+convert.set(9,'.b9');
+var turn = 0;
+var clicked = [0,0,0,0,0,0,0,0,0];
+var over = 0;
+var twop = 0;
+var whowin = 0;
+var bfirst = 0;
+var currbtn = 0;
+var bdecider = 0;
+var bsteps = 0;
+var hsteps = 0;
+var currentm;
+var bmap1 = new Map();
+var bmap2 = new Map();
+var bmap3 = new Map();
+var bmap4 = new Map();
+var bmap5 = new Map();
+var bmap6 = new Map();
+var bmap7 = new Map();
+bmap1.set(4,1); bmap2.set(4,9); bmap3.set(4,3); bmap4.set(4,2); bmap5.set(4,7); bmap1.set(2,1); bmap2.set(2,9); bmap3.set(2,7); bmap4.set(2,4); bmap5.set(2,3); bmap1.set(6,9); bmap2.set(6,1); bmap3.set(6,7); bmap4.set(6,8); bmap5.set(6,3); bmap1.set(8,9); bmap2.set(8,1); bmap3.set(8,3); bmap4.set(8,7); bmap5.set(8,6); bmap1.set(1,4); bmap2.set(1,6); bmap3.set(1,2); bmap4.set(1,8); bmap5.set(1,7); bmap6.set(1,3); bmap7.set(1,9); bmap1.set(3,2); bmap2.set(3,8); bmap3.set(3,6); bmap4.set(3,4); bmap5.set(3,9); bmap6.set(3,1); bmap7.set(3,7); bmap1.set(7,8); bmap2.set(7,2); bmap3.set(7,4); bmap4.set(7,6); bmap5.set(7,1); bmap6.set(7,9); bmap7.set(7,3); bmap1.set(9,8); bmap2.set(9,2); bmap3.set(9,6); bmap4.set(9,4); bmap5.set(9,3); bmap6.set(9,7); bmap7.set(9,1);
+var optdiv = document.querySelector('.opt');
+var griddiv = document.querySelector('.grids');
+document.querySelector('#youbtn').onclick = function() {
+    optdiv.classList.add('hide');
+    griddiv.classList.remove('hide');
+    rediv.classList.remove('hide');
+    currentm = document.querySelector('.youturn');
+    currentm.classList.remove('hide');
+} 
+document.querySelector('#botbtn').onclick = function() {
+    optdiv.classList.add('hide');
+    griddiv.classList.remove('hide');
+    rediv.classList.remove('hide');
+    clicked[4] = 2;
+    currentm = document.querySelector('.youturn');
+    currentm.classList.remove('hide');
+    document.querySelector('.b5').setAttribute('style','background-image:url(\'cross.jpg\')');
+    turn += 1;
+    bfirst = 1;
+}
+document.querySelector('#twobtn').onclick = function() {
+    optdiv.classList.add('hide');
+    griddiv.classList.remove('hide');
+    rediv.classList.remove('hide');
+    currentm = document.querySelector('.p1turn');
+    currentm.classList.remove('hide');
+    twop = 1;
+} 
+function check() {
+    if (clicked[0] === clicked[1] && clicked[1] === clicked[2] && clicked[0] !== 0 && clicked[1] !==0 && clicked[2]!==0) {
```

Comment:
These condition checks are really redundant for our tiny program. If it's already known that `clicked[0] == clicked[1] == clicked[2]` (which is a valid syntax) is `true`, then there's no need to check that the value of indices 0, 1 & 2 non-zero, since any of them can be checked for non-zero value, for the condition to be satisfied.